Makefile

```diff
@@ -1,20 +1,23 @@
-OUTPUT := bin/${current_dir}
+OUTPUT = bin/ecs-deploy
+BUILD_DATE=`date +%FT%T%z`
+GIT_HASH=`git rev-parse HEAD`
+LDFLAGS=-ldflags="-X github.com/gametimesf/log.builtAt=${BUILD_DATE} -X github.com/gametimesf/log.commitHash=${GIT_HASH}"
```

Comment:
can remove this line as we don't use our log package here

Reply:
done. removed that and the ohter code associated with that